plugins/chat/db/post_migrate/20230411023340_update_thread_reply_counts.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class UpdateThreadReplyCounts < ActiveRecord::Migration[7.0]
+  def up
+    DB.exec <<~SQL
+      UPDATE chat_threads threads
+      SET replies_count = subquery.replies_count
+      FROM (
+        SELECT COUNT(*) - 1 AS replies_count, thread_id
+        FROM chat_messages
+        WHERE chat_messages.deleted_at IS NULL AND thread_id IS NOT NULL
```

Comment:
I think we might need an index here to cover this? We have an index on `thread_id` but I believe we will end up with a seq scan to filter for `deleted_at IS NULL`

Reply:
Yes good idea, will add that to Chat::Message :+1: